src/quo2/components/wallet/network_routing/style.cljs

```diff
@@ -0,0 +1,64 @@
+(ns quo2.components.wallet.network-routing.style
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.reanimated :as reanimated]))
+
+(defn container
+  [container-style theme]
+  (assoc container-style
+         :flex-direction   :row
+         :height           64
+         :background-color (colors/theme-colors colors/neutral-100-opa-5 colors/neutral-90 theme)
+         :border-radius    20
+         :overflow         :hidden))
+
+(defn max-limit-bar
+  [{:keys [opacity-shared-value color width]}]
+  (reanimated/apply-animations-to-style
+   {:opacity opacity-shared-value}
+   {:position           :absolute
+    :top                0
+    :bottom             0
+    :left               0
+    :background-color   (str color "10")
+    :width              width
+    :border-right-width 1
+    :border-style       :dashed
+    :border-right-color color
+    :z-index            -1}))
+
+(defn network-bar
+  [{:keys                         [total-amount total-width max-width on-top?
+                                   bar-division? theme]
+    {:keys [color translate-x-shared-value
+            amount-shared-value]} :bar}]
+  (reanimated/apply-animations-to-style
+   {:width     (reanimated/interpolate amount-shared-value [0 total-amount] [0 total-width])
```

Comment:
I think we can just pass the interpolate object instead of creating it here

Reply:
I thought too much on what is more readable, I'll move it :+1: